dev/reference/tools/data-migration/from-aurora.md

```diff
@@ -0,0 +1,197 @@
+  black-white-list: "global"
+  # The configuration item name of mydumper, used to quote the global mydumper configuration. 
+  mydumper-config-name: "global"
+
+-
+  source-id: "mysql-replica-02"
+  black-white-list: "global"
+  mydumper-config-name: "global"
+
+# The global configuration of black and white lists. Each instance can quote it by the configuration item name.
+black-white-list:
+  global:
+    do-tables:                        # The white list of the upstream table to be replicated
+    - db-name: "test_db"              # The database name of the table to be replicated 
+      tbl-name: "test_table"          # The name of the table to be replicated 
+
+# mydumper global configuration. Each instance can quote with the configuration item name. 
+mydumpers:
+  global:
+    extra-args: "-B test_db -T test_table"  # It can only output the `test_table` of `test_db` and configure the parameter of mydumper. 
+```
+
+## Step 5: Start the task
+
+1. Go to dmctl directory `/home/tidb/dm-ansible/resources/bin/`
+
+2. Start dmctl using:
+
+    ```bash
+    ./dmctl --master-addr 172.16.10.71:8261
+    ```
+
+3. Start data replication task using:
+
+    ```bash
+    # `task.yaml` is the previous edited config file 
+    start-task ./task.yaml
+    ```
+    
+    - The task has been started successfully if the returned results do not report any error. 
+    - The permission of upstream Aurora user may not be supported by TiDB if the returned results report the following errors:  
+        
+        ```json
+        {
+            "id": 4,
+            "name": "source db dump privilege chcker",
+            "desc": "check dump privileges of source DB",
+            "state": "fail",
+            "errorMsg": "line 1 column 285 near \"LOAD FROM S3, SELECT INTO S3 ON *.* TO 'root'@'%' WITH GRANT OPTION\" ...",
+            "instruction": "",
+            "extra": "address of db instance - pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com"
+        },
+        {
+            "id": 5,
+            "name": "source db replication privilege chcker",
+            "desc": "check replication privileges of source DB",
+            "state": "fail",
+            "errorMsg": "line 1 column 285 near \"LOAD FROM S3, SELECT INTO S3 ON *.* TO 'root'@'%' WITH GRANT OPTION\" ...",
+            "instruction": "",
+            "extra": "address of db instance - pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com"
+        }
+        ```
+        
+        Using either of the following solutions, you can use `start-task` to restart the task:
+        1. Remove the unnecessary permission not supported by TiDB for Aurora user when migrating data 
+        2. If Aurora user is ensured to have the permission that DM needs, then add the following configuration item in `task.yaml` config file to skip the permission precheck when start the task. 
+            ```yaml
```

Comment:
Leave a blank line before this code block. The format is also wrong in docs-cn and you can check the wrong display at the PingCAP website [here](https://pingcap.com/docs-cn/tools/dm/from-aurora/). @huangxiuyan Please help update it too.

```suggestion

            ```yaml
```